### PR TITLE
[ADM-240] Extract methods in GenerateSprintReporterService.ts to Jira.ts

### PR DESCRIPTION
**Pull request by JDTheBuried**
[ADM-240](https://dorametrics.atlassian.net/jira/software/projects/ADM/boards/2?selectedIssue=ADM-240)

### CODE REVIEW

**Review by chaoyangyes — APPROVED** [review]
LGTM

File: backend/src/models/kanban/SprintStatistics.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export class SprintStatistics {
-  public completedCardsNumber?: Array<CompletedCardNumber>;
+  public completedCardNumber?: Array<CompletedCardNumber>;
```
Comment (WenZhaosen) [inline backend/src/models/kanban/SprintStatistics.ts:2]: `completedCardNumber` seems like just one number , `completedCardNumberList` may be clearer.
And param in constructor is not renamed.

File: backend/src/services/GenerateReporter/GenerateReportService.ts [hunk 2]
```diff
@@ -278,7 +277,7 @@ export class GenerateReportService {
       kanbanSetting.token,
       kanbanSetting.site
     );
-    this.cards = await kanban.getStoryPointsAndCycleTime(
+    const model: StoryPointsAndCycleTimeRequest =
```
Comment (WenZhaosen) [inline backend/src/services/GenerateReporter/GenerateReportService.ts:280]: Need a more meaningful name.

File: backend/src/services/kanban/Linear/Linear.ts [hunk 3]
```diff
@@ -190,4 +191,11 @@ export class Linear implements Kanban {
       ...stateChanged,
     ];
   }
+
+  public async getSprintStatistics(
+    model: StoryPointsAndCycleTimeRequest,
+    cards: Cards
+  ): Promise<SprintStatistics> {
+    return new SprintStatistics();
```
Comment (WenZhaosen) [inline backend/src/services/kanban/Linear/Linear.ts:199]: Can we throw an error here? like 'Linear sprint statistics service not available' or some other message.

**Review by WenZhaosen — APPROVED** [review]

**Pull request merged**